main.py

```diff
@@ -182,27 +191,27 @@ async def on_member_remove(member: discord.Member):
 @bot.event
 async def on_ready():
     drone_dao.add_new_drone_members(bot.guilds[0].members)
-    global checking_for_completed_orders, reporting_storage, checking_for_stored_drones_to_release, updating_status_message, checking_for_elapsed_timers
+    global checking_for_completed_orders, reporting_storage, checking_for_stored_drones_to_release, checking_for_elapsed_timers
```

Comment:
Is this line and the booleans still required? Could we also ask the Task objects if they are running to get the same result?